### ISSUE TITLE
jsitor SSL certificate is expired on Feb 6 2023

### ISSUE THREAD

**Issue by Reza-1** [jsitor/jsitor#121]
https://jsitor.com/

Peer’s Certificate has expired.

HTTP Strict Transport Security: true
HTTP Public Key Pinning: false

**Comment by ashvin777** [comment]
Yes we are going to fix the issue with few minutes. Thanks for reporting.
